extension/src/extension.ts

```diff
@@ -22,12 +22,13 @@ import * as vscode from 'vscode';
 import { Commands, Configs, Constants } from './commands';
 import { JUnitCodeLensProvider } from './junitCodeLensProvider';
 import { TestResourceManager } from './testResourceManager';
-import { OutputChannel, SnippetString, ExtensionContext, window } from 'vscode';
+import { OutputChannel, ExtensionContext, Uri } from 'vscode';
```

Comment:
Already import the vscode as *.  This import is not necessary.

Reply:
I added this to not introduce `vscode.` frequently. Removing `windows` as it is lowercase and might be confused with local variables. I think it is good to keep consistent. Which way do you think is better? `import * as vscode from vscode` or import all the used type in vscode module explicitly

Reply:
Either import as * or import separate. Either way works but not mixed.